sources/packages/backend/apps/api/src/services/application-restriction-bypass/application-restriction-bypass.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from "@nestjs/common";
+import {
+  ApplicationRestrictionBypass,
+  ApplicationStatus,
+  RecordDataModelService,
+} from "@sims/sims-db";
+import { ApplicationRestrictionBypassSummaryAPIOutDTO } from "apps/api/src/route-controllers/application-restriction-bypass/models/application-restriction-bypass.dto";
+import { DataSource } from "typeorm";
+
+/**
+ * Service layer for application restriction bypasses.
+ */
+@Injectable()
+export class ApplicationRestrictionBypassService extends RecordDataModelService<ApplicationRestrictionBypass> {
+  constructor(dataSource: DataSource) {
+    super(dataSource.getRepository(ApplicationRestrictionBypass));
+  }
+
+  /**
+   * Returns all restriction bypasses for a given application.
+   * @param applicationId id of the application to retrieve restriction bypasses.
+   * @returns application restriction bypasses.
+   */
+  async getApplicationRestrictionBypasses(
+    applicationId: number,
+  ): Promise<ApplicationRestrictionBypassSummaryAPIOutDTO[]> {
+    return this.repo
+      .createQueryBuilder("applicationRestrictionBypass")
+      .select("applicationRestrictionBypass.id", "id")
+      .addSelect("applicationRestrictionBypass.createdAt", "createdAt")
+      .addSelect("restriction.restrictionType", "restrictionType")
+      .addSelect("restriction.restrictionCode", "restrictionCode")
+      .addSelect("studentRestriction.isActive", "isActive")
+      .innerJoin(
```

Comment:
Are we not showing the `Restriction Management` tab at all when there are no restrictions and/or their bypasses?